src/mscorlib/shared/System/Char.cs

```diff
@@ -274,11 +274,7 @@ private static bool IsWhiteSpaceLatin1(char c)
             // U+000d = <control> CARRIAGE RETURN
             // U+0085 = <control> NEXT LINE
             // U+00a0 = NO-BREAK SPACE
-            if ((c == ' ') || (c >= '\x0009' && c <= '\x000d') || c == '\x00a0' || c == '\x0085')
-            {
-                return (true);
-            }
-            return (false);
+            return c == ' ' || (c >= '\x0009' && c <= '\x000d') || c == '\x00a0' || c == '\x0085';
```

Comment:
Probably you should force inline this method if it is not being inlined in all call-sites. This internal method calls end up showing in profile runs.